{"category": "ham", "to_address": "sugar@laptop.org", "from_address": "\"Manuel Aguilar\" <manuel.aguilar.alvarez@gmail.com>", "subject": "[sugar] Can't start sugar", "body": "Hey folks,I don't have something clear when I try ./sugar-jhbuild run:- The sugar shell tries to use a socket located at /var/run/dbus/system_bus_socket. My distro (Mandriva 2007.0) didn't provide it with that name. So I created a symlink to /var/run/dbus/system_dbus_socket\n- After retrying I was able to see the sugar pointer (the big arrow) but then I got a second exception, here is the traceback:Traceback (most recent call last):\u00a0 File \"/home/manuel/videos/2007-05-28-SUGAR/build/bin/sugar-shell\", line 117, in ?\n\u00a0\u00a0\u00a0 model = ShellModel()\u00a0 File \"/home/manuel/videos/2007-05-28-SUGAR/build/share/sugar/shell/model/ShellModel.py\", line 46, in __init__\u00a0\u00a0\u00a0 self._pservice = presenceservice.get_instance()\u00a0 File \"/home/manuel/videos/2007-05-28-SUGAR/build/lib/python2.4/site-packages/sugar/presence/presenceservice.py\", line 482, in get_instance\n\u00a0\u00a0\u00a0 _ps = PresenceService(allow_offline_iface)\u00a0 File \"/home/manuel/videos/2007-05-28-SUGAR/build/lib/python2.4/site-packages/sugar/presence/presenceservice.py\", line 82, in __init__\u00a0\u00a0\u00a0 self._get_ps()\n\u00a0 File \"/home/manuel/videos/2007-05-28-SUGAR/build/lib/python2.4/site-packages/sugar/presence/presenceservice.py\", line 122, in _get_ps\u00a0\u00a0\u00a0 raise RuntimeError(\"Failed to connect to the presence service.\")\nRuntimeError: Failed to connect to the presence service.Is it relevant to have sugar's dbus-daemon running at startup or it should be ok with the host system's ?Manuel.\n"}